packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -59,6 +59,13 @@ type ChatMessageComponentProps = {
    * @param userId - user Id
    */
   onRenderAvatar?: OnRenderAvatarCallback;
+
+  /* @conditional-compile-remove(dateTimeCustomization) */
```

Comment:
`ChatMessageComponentProps` is not exported from this module (and from the public API).

So there is no need for any conditional compilation here. Please add it unconditionally (this also simplifies your implementation).

Only public API needs to be added conditionally.